src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -269,6 +278,8 @@ public function createAction(Request $request)
             return $this->viewHandler->handle($configuration, View::create($form, Response::HTTP_BAD_REQUEST));
         }
 
+        $this->eventDispatcher->dispatch(ResourceActions::INITIALIZE_CREATE, $configuration, $newResource);
```

Comment:
We have ``dispatchPre``, ``dispatchPost`` methods on the dispatcher, shouldn't this be ``dispatchInitialize`` and use ``ResourceActions::CREATE``?